apps/web/src/scenes/ContentPages/PostsFeaturePage.tsx

```diff
@@ -0,0 +1,159 @@
+import Head from 'next/head';
+import { useCallback } from 'react';
+import styled from 'styled-components';
+
+import breakpoints, { pageGutter } from '~/components/core/breakpoints';
+import { ButtonLink } from '~/components/core/Button/Button';
+import Markdown from '~/components/core/Markdown/Markdown';
+import { VStack } from '~/components/core/Spacer/Stack';
+import { TitleCondensed, TitleDiatypeL } from '~/components/core/Text/Text';
+import { useTrack } from '~/shared/contexts/AnalyticsContext';
+import colors from '~/shared/theme/colors';
+
+import { CmsTypes } from './cms_types';
+import Faq from './ContentModules/Faq';
+import FeatureHighlight from './ContentModules/FeatureHighlight';
+
+type Props = {
+  pageContent: CmsTypes.FeaturePage;
+};
+
+export default function PostsFeaturePage({ pageContent }: Props) {
+  const track = useTrack();
+  const handleGetStartedClick = useCallback(() => {
+    track('Posts Feature Page: Clicked Get Started');
+  }, [track]);
```

Comment:
this will be done directly on the button soon :)

Reply:
yess so amazing